plugin/build.gradle

```diff
@@ -51,6 +51,14 @@ dependencies {
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
     implementation group: 'org.apache.commons', name: 'commons-math3', version: '3.6.1'
     implementation "org.apache.logging.log4j:log4j-slf4j-impl:2.19.0"
+    // implementation group: 'software.amazon.awssdk', name: 'sagemaker', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'sagemakerruntime', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'auth', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'sdk-core', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'annotations', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'aws-core', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'regions', version: '2.18.21'
+    implementation group: 'software.amazon.awssdk', name: 'utils', version: '2.18.21'
     testImplementation group: 'commons-io', name: 'commons-io', version: '2.11.0'
```

Comment:
It is true that in this scenario we can use our own company sdk package. But if we want to access other third parties, we should not rely on their sdk packages directly in the ml-common code. This would involve the question of whether the code is allowed to be used between companies, has this been considered before?